### ISSUE TITLE
Graph2d: Event that fires when the user drags the timeline

### ISSUE THREAD

**Issue by Gillingham** [almende/vis#309]
Right now we have the rangechange and rangechanged events that fire when the user drags the timeline, OR a function like fit() is called.

Ideally I would be able to bind to an event that only fired when the user actually drags the timeline.

My use case for this is I have a toggle-able autozoom capability that calls fit() every time we add data via ajax if enabled, and I want to keep that enabled but toggle it to off only when the user manually toggles it  OR they drag the timeline manually. Right now I can't do this because when I call fit() after adding new items the rangechange(d) events get triggered.

**Comment by AlexDM0** [comment]
Hi,

Over the last year a lot of feature requests have been made. We have just introduced our new website which has a list of the requested features. We have placed this request on that list.

The list can be found here:
http://visjs.org/featureRequests.html

An explaination of the new system can be found here:
http://visjs.org/blog.html#New\ website\ for\ vis.js!

I would like to stress that this does not mean we abandon this request. Discussion here will continue if needed on this feature but we will close it to keep our Github issue page more of a bug-todo list.

Future feature requests will still be made here and then added to the website by us.

Regards,

Alex
